### ISSUE TITLE
SQL injection security risk in request url

### ISSUE THREAD

**Issue by llhhbc** [grafana/grafana#68048]
**Steps to reproduce**:
![image](https://user-images.githubusercontent.com/6076617/236973973-ac345356-c00a-452e-81da-e9f73d345aae.png)

**Actual Result**:
Query OK

**Expected Result**
Should return request parameter error

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version: 6.5.1
- Data source type & version:
- User OS & Browser: 
- Others:

**Comment by sakjur** [comment]
👋 This is a documented design property of Grafana that is mentioned both in the docs and when you're configuring an SQL based data source, see https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/#limit-viewer-query-permissions

Beyond that, Grafana 6.5 hasn't been supported since 2020.

**Comment by llhhbc** [comment]
@sakjur Thank you for your reply.

Access control is already possible, but if this kind of query parameter sql injection is avoided, it cannot be realized yet. Because the vulnerability scanning tool thinks that the interface injection returns successfully, it is considered risky, although it does not necessarily have business risks
